app/mainAppWindow/index.js

```diff
@@ -117,6 +117,39 @@ function applyAppConfiguration(config, window) {
 		});
 	}
 
+	if (config.optInV2) {
+		config.url = 'https://teams.microsoft.com/v2/';
+		window.webContents.executeJavaScript('localStorage.getItem("tmp.isOptedIntoT2Web");', true)
+			.then(result => {
+				if ((result == null) || !result) {
+					window.webContents.executeJavaScript('localStorage.setItem("tmp.isOptedIntoT2Web", true);', true)
+						.then(window.reload())
+						.catch(err => {
+							console.log('could not set localStorage variable', err);
+						});
+				}
+			})
+			.catch(err => {
+				console.log('could not read localStorage variable', err);
+			});
+	}
+
+	if (!config.optInV2) {
```

Comment:
Simpler for this to just be an else
```suggestion
	} else {
```